src/modules/app/route-configs/index.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import type { Route } from 'redux-first-router'
+import { RouteConfigInterface } from './RouteConfigInterface'
+import CategoriesRouteConfig, { CATEGORIES_ROUTE } from './CategoriesRouteConfig'
+import I18nRedirectRouteConfig, { I18N_REDIRECT_ROUTE } from './I18nRedirectRouteConfig'
+import LandingRouteConfig, { LANDING_ROUTE } from './LandingRouteConfig'
+import MainDisclaimerRouteConfig, { MAIN_DISCLAIMER_ROUTE } from './MainDisclaimerRouteConfig'
+import EventsRouteConfig, { EVENTS_ROUTE } from './EventsRouteConfig'
+import WohnenRouteConfig, { WOHNEN_ROUTE } from './WohnenRouteConfig'
+import ExtrasRouteConfig, { EXTRAS_ROUTE } from './ExtrasRouteConfig'
+import SearchRouteConfig, { SEARCH_ROUTE } from './SearchRouteConfig'
+import PoisRouteConfig, { POIS_ROUTE } from './PoisRouteConfig'
+import DisclaimerRouteConfig, { DISCLAIMER_ROUTE } from './DisclaimerRouteConfig'
+import SprungbrettRouteConfig, { SPRUNGBRETT_ROUTE } from './SprungbrettRouteConfig'
+
+export const LocationLayoutRoutes = [CATEGORIES_ROUTE, EVENTS_ROUTE, EXTRAS_ROUTE, SPRUNGBRETT_ROUTE, WOHNEN_ROUTE,
+  DISCLAIMER_ROUTE, SEARCH_ROUTE, POIS_ROUTE]
+
+export const routeConfigs: {[string]: RouteConfigInterface<any, any>} = {
```

Comment:
do we need to export this? Also these shouldn't be anys (try to use *)

Reply:
We don't have to export this, but * does not work. * would mean every route config has the same generic types, which is not the case.